src/git.rs

```diff
@@ -10,8 +10,9 @@
 
 use cargo::util::{CargoResult, CargoResultExt};
 use cargo::Config;
-use failure::err_msg;
+use anyhow::anyhow;
 use git2::{self, Repository};
+use lazy_static::lazy_static;
```

Comment:
this is not needed for 1.48 but if you want to cleanup you should probably do this and removing all uses of`extern crate` in a separate commit.

Reply:
Not really doing cleanup of extern crate here just wanted to avoid two different ways of importing macros, since before `macro_use` was used to import the macros